libvast/vast/system/exporter.hpp

```diff
@@ -32,6 +32,8 @@
 namespace vast::system {
 
 struct exporter_state {
+  caf::settings status();
+
```

Comment:
Could it make sense to use a free function in the implementation file instead? The struct has only state, and then using `caf::settings` would not show up in the header.

Reply:
What's the issue with this "showing up"? It's just a forward declaration. Having a member function feels the most natural to me. Also, this is simply modeled after other actors such as the INDEX.

Reply:
It's not really a problem, I was just scratching my head as we were extending a struct with only data so far.